src/Components/Quiz/Quiz.js

```diff
@@ -54,13 +62,15 @@ const Quiz = ({ questions, handleQuizResults}) => {
         <div className="questionDiv">
             <h2 className="question">{questions[counter].question}</h2>
             <p className="questionCategory">{questions[counter].category}</p>
-            <div className="buttonDiv">
+           <div className="buttonDiv">
                 <button className="true" onClick={(event) => checkTrueCorrect(event)}>TRUE</button>
                 <button className="false" onClick={(event) => checkFalseCorrect(event)}>FALSE</button>
             </div>
             <div className="bottomDiv">
                 <p className="progress">{counter}/10</p>
-                <button className="nextQuestionButton" onClick={increaseCounter}>Next Question</button>
+                {isClicked &&
```

Comment:
Nice implementation of the isClicked hook! Looks like a good way to go through questions reliably.